DigitalLearningSolutions.Web.Tests/Controllers/Register/RegisterControllerTests.cs

```diff
@@ -34,57 +35,114 @@ public void Setup()
+                    userService,
+                    customPromptHelper
+                )
                 .WithDefaultContext()
                 .WithMockTempData();
         }
 
         [Test]
-        public void IndexPost_with_existing_user_for_centre_fails_validation()
+        public void PersonalInformationPost_with_existing_user_for_centre_fails_validation()
         {
             // Given
             var duplicateUser = UserTestHelper.GetDefaultDelegateUser();
-            var model = new RegisterViewModel
+            var model = new PersonalInformationViewModel
             {
                 FirstName = "Test",
                 LastName = "User",
                 Centre = duplicateUser.CentreId,
                 Email = duplicateUser.EmailAddress
             };
             A.CallTo(() => userService.GetUsersByEmailAddress(duplicateUser.EmailAddress!))
-                .Returns((null, new List<DelegateUser> {duplicateUser}));
+                .Returns((null, new List<DelegateUser> { duplicateUser }));
 
             // When
-            var result = controller.Index(model);
+            var result = controller.PersonalInformation(model);
 
             // Then
             A.CallTo(() => userService.GetUsersByEmailAddress(duplicateUser.EmailAddress!)).MustHaveHappened();
             result.Should().BeViewResult().WithDefaultViewName();
         }
 
         [Test]
-        public void IndexPost_with_existing_user_for_different_centre_is_allowed()
+        public void PersonalInformationPost_with_existing_user_for_different_centre_is_allowed()
         {
             // Given
             controller.TempData.Set(new DelegateRegistrationData());
             var duplicateUser = UserTestHelper.GetDefaultDelegateUser();
-            var model = new RegisterViewModel
+            var model = new PersonalInformationViewModel
             {
                 FirstName = "Test",
                 LastName = "User",
                 Centre = duplicateUser.CentreId + 1,
                 Email = duplicateUser.EmailAddress
             };
             A.CallTo(() => userService.GetUsersByEmailAddress(duplicateUser.EmailAddress!))
-                .Returns((null, new List<DelegateUser> {duplicateUser}));
+                .Returns((null, new List<DelegateUser> { duplicateUser }));
 
             // When
-            var result = controller.Index(model);
+            var result = controller.PersonalInformation(model);
 
             // Then
             A.CallTo(() => userService.GetUsersByEmailAddress(duplicateUser.EmailAddress!)).MustHaveHappened();
             result.Should().BeRedirectToActionResult().WithActionName("LearnerInformation");
         }
+
+        [Test]
+        public void IndexPost_with_invalid_centreId_param_shows_error()
+        {
+            // Given
+            controller.TempData.Set(new DelegateRegistrationData());
+            var centreId = CentreTestHelper.GetDefaultCentre().CentreId;

```

Comment:
Do we need to get the id off a default centre or can it be any int given that we're mocking the centresDataService anyway?